db/migrate/20220902150227_create_regular_transactions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRegularTransactions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :regular_transactions, id: :uuid do |t|
+      t.references :legal_aid_application, null: false, foreign_key: { on_delete: :cascade }, index: false, type: :uuid
```

Comment:
❓ I assume we do not have any callbacks on regular transactions that would be missed by a cascade delete here or would the `has_many :regular_transactions, dependant: :destroy` trigger those in any event?

Reply:
From my perspective it didn't make sense for a regular transaction to exist if it isn't associated to a legal aid application in any circumstance.

If a `legal_aid_application` record is deleted from the database, any `regular_transaction` records that reference that `legal_aid_application_id` are also deleted.

This cascades deletions at a database level, but I wouldn't really expect us to be deleting records this way normally.

Typically, we'd be destroying records with ActiveRecord and the database constraint wouldn't come into play.